core/src/test/java/org/mule/routing/DefaultRouterResultsHandlerTestCase.java

```diff
@@ -130,7 +133,8 @@ public void aggregateMultipleEvents() throws Exception
         // Because a new MuleMessageCollection is created, propagate properties from
         // original event
         // TODO DF
-        // assertEquals("value1", result.getMessage().getInvocationProperty("key1"));
+        assertEquals("value1", result.getMessage().getInvocationProperty("key1"));
+        assertEquals(simpleDateType1.getMimeType(), result.getFlowVariableDataType("key1").getMimeType());
```

Comment:
you should not assert just over the mimeType. You should rely on the DataType's equals(Object) method